### DISCUSSION TITLE
Username not updating in client overview

### DISCUSSION THREAD

**Discussion by AWHarvard2** [rustdesk/rustdesk#8031] [General]
### Bug Description

The username firstname.lastname@hostname in the client overview is never updating.
We install all clients by our SCCM tool to the client page will alwasys show the sccm's user account sccm@PC11111 as the user of a device. This breaks the search by user filter for us.

### How to Reproduce

Just look into the clients page and change the loggen in user on one client.

### Expected Behavior

Username updating according to the user currently logged on into the Windows client

### Operating system(s) on local side and remote side

Windows 11 -> Windows 11

### RustDesk Version(s) on local side and remote side

latest

### Screenshots

No screenshot needed

**Comment by rustdesk** [comment]
@21pages follow up. Login into windows with different account to see if username changed on devices page of web console. Installed windows version.

**Reply by AWHarvard2** [reply]
Tested it with 3 different Windows clients, username didn't change in the console also not after hours.

**Comment by rustdesk** [comment]
We had a fix for this https://github.com/rustdesk/rustdesk/commit/78d7bfac01d7491e1d1a35050f4569963f2ed025, ready in latest night build and custom client.
